study.md

```diff
@@ -43,5 +59,13 @@ your deployed Ember app? What do you need if you want to use the `history` API
 instead of `hash` for `location`?
 
 ```md
-<!-- your answer here -->
+Hash URLs are used by javascript to load a page without fully connecting to the
+server. In GitHub pages you need to be aware of how you reference images there
+are specific formats to how you list it in CSS and embed into templates. The
+you want to set up your environment.js file.
+
+Not going to lie still a little be confused on how ember doesn't break the
+internet.  Because in my mind I see ember doing almost the same thing as the
+hash except the only difference is it uses the router to maintain url integrity.
+Maybe this will be covered more in class.
```

Comment:
Hash URLs are necessary in order for Ember to work on GitHub Pages because GitHub pages isn't running an Ember server, just a static page server. It can't respond to other URLs and dynamically generate pages, it can only load pages that are hard-coded.  Fortunately, though, Ember can also be run in `hash` routing mode, which allows it to split out the second half of the URL and load the appropriate templates without loading a new page.